nginx.conf

```diff
@@ -0,0 +1,51 @@
+events {
```

Comment:
Could you please move this to a `deploy` directory? It probably doesn't hurt if we commit an example nginx config, but let's not put it into the root. And let's also modify the default path in `.env.example` to point to the `deploy` directory.

Reply:
Done